modules/nextflow/src/main/groovy/nextflow/k8s/K8sTaskHandler.groovy

```diff
@@ -216,6 +218,14 @@ class K8sTaskHandler extends TaskHandler implements FusionAwareTask {
             .withLabels(getLabels(task))
             .withAnnotations(getAnnotations())
             .withPodOptions(getPodOptions())
+            .withHostMount("/opt/latch-env", "/opt/latch-env")
+
+        builder.withEnv(PodEnv.value("LATCH_NO_CRASH_REPORT", "1"))a
+
+        if (System.getenv("LATCH_NF_DEBUG") != "true") {
```

Comment:
the k8s executor would break anyway if we were running this locally right?

Reply:
yeah probably